ts/features/idpay/payment/screens/IDPayPaymentResultScreen.tsx

```diff
@@ -1,34 +1,80 @@
 import { useSelector } from "@xstate/react";
+import * as O from "fp-ts/lib/Option";
+import { pipe } from "fp-ts/lib/function";
 import { default as React } from "react";
-import { SafeAreaView, View } from "react-native";
-import { ContentWrapper } from "../../../../components/core/ContentWrapper";
+import { SafeAreaView, StyleSheet, View } from "react-native";
+import {
+  IOPictograms,
+  Pictogram
+} from "../../../../components/core/pictograms";
+import { VSpacer } from "../../../../components/core/spacer/Spacer";
 import { Body } from "../../../../components/core/typography/Body";
+import { NewH4 } from "../../../../components/core/typography/NewH4";
 import { IOStyles } from "../../../../components/core/variables/IOStyles";
 import FooterWithButtons from "../../../../components/ui/FooterWithButtons";
+import I18n from "../../../../i18n";
+import { PaymentFailure, PaymentFailureEnum } from "../xstate/failure";
 import { usePaymentMachineService } from "../xstate/provider";
-import { selectIsFailure, selectTransactionData } from "../xstate/selectors";
+import { selectFailureOption } from "../xstate/selectors";
+
+type FailureContent = {
+  title: string;
+  subtitle?: string;
+  icon: IOPictograms;
+};
+type PaymentResultsType = PaymentFailure | "SUCCESS";
```

Comment:
```suggestion
type PaymentResultType = PaymentFailure | "SUCCESS";
```
Use the singular when defining types.

Reply:
adressed in https://github.com/pagopa/io-app/pull/4682/commits/3cff96b19bae16d42fd7ae3071da832a6a865220